tests/e2e/Services/Databases/DatabasesConsoleClientTest.php

```diff
@@ -50,6 +51,38 @@ public function testCreateCollection(): array
         $this->assertEquals(201, $movies['headers']['status-code']);
         $this->assertEquals($movies['body']['name'], 'Movies');
 
+        /**
+         * Test When database is disabled but can still create collections
+         */
+        $database = $this->client->call(Client::METHOD_PUT, '/databases/' . $databaseId, [
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ], [
+            'name' => 'invalidDocumentDatabase Updated',
+            'enabled' => false,
+        ]);
+
+        $this->assertFalse($database['body']['enabled']);
+
+        $movies = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'collectionId' => ID::unique(),
+            'name' => 'TvShows',
+            'permissions' => [
+                Permission::read(Role::any()),
+                Permission::create(Role::any()),
+                Permission::update(Role::any()),
+                Permission::delete(Role::any()),
+            ],
+            'documentSecurity' => true,
+        ]);
+
```

Comment:
Let's add assertions for list/get/update/delete collection in console mode when the database is disabled as well

Reply:
Done